pkg/storage/mock/storer.go

```diff
@@ -0,0 +1,35 @@
+package mock
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ethersphere/bee/pkg/storage"
+)
+
+type mockStorer struct {
+	store map[string][]byte
+}
+
+func NewStorer() storage.Storer {
+	s := &mockStorer{
+		store: make(map[string][]byte),
+	}
+
+	return s
+}
+
+func (m *mockStorer) Get(ctx context.Context, addr []byte) (data []byte, err error) {
+	k := fmt.Sprintf("%x", addr)
+	v, has := m.store[k]
+	if has {
+		return v, nil
```

Comment:
I think that if check should be reversed. Positive outcome should be at the end of the function, and errors should be in the branches.